common/components/ui/OfflineSymbol.tsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+import React from 'react';
+import wifiOn from 'assets/images/wifi-on.svg';
+import wifiOff from 'assets/images/wifi-off.svg';
+
+type sizeType = 'small' | 'medium' | 'large';
```

Comment:
This is obvious duplicate code, but I'm not sure how much I want to pull this into a HOC yet.